src/service-runner-config.json

```diff
@@ -21,7 +20,13 @@
             ],
             "stop": [],
             "teardown": []
-          }
+          },
+          "health": {
+           "interval": 5000, 
+           "retries": 2,
+           "port": "${DYNAMIC_TCP_PORT_1}",
+           "proto": "tcp"
```

Comment:
can we use the full name, `protocol` ?

Reply:
sure

Reply:
I'll refactor this in the next PR with healthchecks